src/cowpatch/base_elements.py

```diff
@@ -210,7 +214,14 @@ def __add__(self, other):
             # combine with layout -------------
             self.__layout = other
         elif inherits(other, annotation):
-            raise ValueError("currently not implimented addition with annotation")
+            if self.annotation is None:
+                other_copy = copy.deepcopy(other)
+                other_copy._clean_up_attributes()
```

Comment:
the _clean_up_attributes allows special overriding & emptying of values when we add different `annotation` classes together